app/controllers/adventures_controller.rb

```diff
@@ -3,7 +3,7 @@ class AdventuresController < ApplicationController
   load_permissions_and_authorize_resource
 
   def index
-    @adventure = @adventures.published.first
+    @adventure = @introduction.adventures.published.first
```

Comment:
Does this make the index page only showing the adventures from the current introduction? If yes, maybe reconsider since one might want to look at old adventures as well.